third_party/2and3/decorator.pyi

```diff
@@ -0,0 +1,90 @@
+# Stubs for decorator (Python 2 and 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+import sys
+from typing import Any, Callable, Dict, List, NamedTuple, Optional, Pattern, Text, Tuple, TypeVar
+
+_C = TypeVar("_C", bound=Callable[..., Any])
+_Func = TypeVar("_Func", bound=Callable[..., Any])
+_Caller = TypeVar("_Caller", bound=Callable[..., Any])
+
+def get_init(cls): ...
+
+# Duplicated from `stdlib/3/inspect.pyi`
+FullArgSpec = NamedTuple('FullArgSpec', [('args', List[str]),
+                                         ('varargs', Optional[str]),
+                                         ('varkw', Optional[str]),
+                                         ('defaults', tuple),
+                                         ('kwonlyargs', List[str]),
+                                         ('kwonlydefaults', Dict[str, Any]),
+                                         ('annotations', Dict[str, Any]),
+                                         ])
+
+if sys.version_info >= (3,):
+    iscoroutinefunction: Callable[[Callable[..., Any]], bool] = ...  # really inspect.iscoroutinefunction
+    getfullargspec: Callable[[object], FullArgSpec] = ...  # really inspect.getfullargspec
+else:
+    def iscoroutinefunction(f: Callable[..., Any]) -> bool: ...
+    def getfullargspec(func: object) -> FullArgSpec: ...
+
+if sys.version_info >= (3, 2):
+    from contextlib import _GeneratorContextManager
+else:
+    from contextlib import GeneratorContextManager as _GeneratorContextManager
+
+DEF: Pattern[Text]
+
+class FunctionMaker:
+    args: List[Text] = ...
+    varargs: Optional[Text] = ...
+    varkw: Optional[Text] = ...
+    defaults: Tuple[Any] = ...
+    kwonlyargs: List[Text] = ...
+    kwonlydefaults: Optional[Text] = ...
+    shortsignature: Optional[Text] = ...
+    name: Text = ...
+    doc: Optional[Text] = ...
+    module: Optional[Text] = ...
+    annotations: Dict[Text, Any] = ...
+    signature: Optional[Text] = ...
```

Comment:
All paths setting `signature` will lead to a not-`None` signature. There's also a `shortsignature` field.

```suggestion
    signature: Text
    shortsignature: Optional[Text]
```

Reply:
shortsignature is on line 45. is there a standard for laying this out? perhaps alphabetical order would make it easier to find things?